bootstrap/api/v1alpha1/rke2configtemplate_webhook.go

```diff
@@ -36,39 +36,39 @@ func (r *Rke2ConfigTemplate) SetupWebhookWithManager(mgr ctrl.Manager) error {
 
 //+kubebuilder:webhook:path=/mutate-bootstrap-cluster-x-k8s-io-v1alpha1-rke2configtemplate,mutating=true,failurePolicy=fail,sideEffects=None,groups=bootstrap.cluster.x-k8s.io,resources=rke2configtemplates,verbs=create;update,versions=v1alpha1,name=mrke2configtemplate.kb.io,admissionReviewVersions=v1
 
-var _ webhook.Defaulter = &Rke2ConfigTemplate{}
+var _ webhook.Defaulter = &RKE2ConfigTemplate{}
 
 // Default implements webhook.Defaulter so a webhook will be registered for the type
-func (r *Rke2ConfigTemplate) Default() {
-	rke2configtemplatelog.Info("default", "name", r.Name)
+func (r *RKE2ConfigTemplate) Default() {
+	RKE2configtemplatelog.Info("default", "name", r.Name)
 
 	// TODO(user): fill in your defaulting logic.
 }
 
 // TODO(user): change verbs to "verbs=create;update;delete" if you want to enable deletion validation.
```

Comment:
nit, we can remove these todos that were left in as part of scaffolding. But this can be done later.

Reply:
Done!